src/main/kotlin/com/workos/organizations/OrganizationsApi.kt

```diff
@@ -14,12 +14,39 @@ import com.workos.organizations.models.OrganizationList
  * The OrganizationsApi provides convenience methods for working with WorkOS Organizations.
  */
 class OrganizationsApi(private val workos: WorkOS) {
+  /**
+   * Options when setting the domains for an organization.
```

Comment:
Nit: hen I built this I tried to not have any class/enum inlined into another, and have them all separated, within the `types` folder. I think it makes of a cleaner, easier way to find and reuse things. But I also understand this is more compact.

Reply:
Makes sense! I'll move it there.